lib/tasks/combined_publisher_schemas.rake

```diff
@@ -0,0 +1,21 @@
+require 'govuk_content_schemas/schema_combiner'
+require 'json-schema'
+require 'json'
+
+READER = JSON::Schema::Reader.new(accept_file: true, accept_uri: false)
+
+rule %r{^dist/formats/.*/publisher/schema.json} => ->(f) {
+    Rake::FileList.new(
+      "formats/{definitions,metadata,v1_metadata,base_links}.json",
+      f.pathmap("%{^dist/,}p").pathmap("%d/{details,links}.json")
+    )
+  } do |t|
```

Comment:
I think this could do with some explaining or variable extraction, I'm having a little bit of trouble grasping what is going on here :)